website/docs/addon_applications_admin.md

```diff
@@ -1,63 +1,294 @@
-![settings_applications](assets/settings/settings_addapplication.gif)
+#### Built-in Applications
+Each application definition has an application group e.g. Maya, Houdini, Nuke and etc.
+
+![settings_applications](assets/applications/application_grorup.png)
+
+Each Application is made of two levels.
+1. **Application group:**
+   1. **Host name:** Pipeline integration name supplied by an AYON addon.
+   2. **Environment:** Define extra environments that are applicable to all versions of the given application version dependent.
+2. **Application variant:** For each application variant we have
+   1. **Name:** application variant name
+   2. **Label:** application variant label
+   3. **Executables:** Define executables (per platform).
+   ![](assets/applications/mutliplatform_executables.png)
+   4. **Arguments:** Define default arguments (`--nukex` for instance)
+   ![](assets/applications/mutliplatform_args.png)
+   5. **Environment:** Extend application's main environment.
+   6. **+ button:** To add more variants.
+
+:::info Naming Conventions for Names and Labels
+
+We currently allow the use of any characters. 
+However, we adhere to a specific naming convention alongside the default settings. 
+For names, we typically use lowercase letters, numbers, and hyphens (`-`).
+For labels, both uppercase and lowercase letters, as well as periods (`.`), are used.
+
+:::
+
+#### Additional Applications
+
+It's possible to use applications addon to launch any applications that are not in built-in integrations that come with AYON.
+
+:::note
+Additional applications will only _launch_ but offer no further AYON integration (e.g. menus, creators, publishers) as that requires an AYON addon for that application, at which point it should not be under _additional applications_ anymore.
+:::
+
+![](assets/applications/additional_applications_1.png)
+- **+ button:** Add application group.
+
+Application group in additional applications exposes few more attributes:
+
+![](assets/applications/additional_applications_2.png)
+
+- **Application Name:** The name of the application.
+- **Application Label:** The label of the application and the application group (the name next to the toggle).
+- **Application Icon:** The icon that would be displayed by the AYON launcher.
+
+:::tip Application Icon
+<!-- TODO: Add more info about what ayon expects regards the application icon. -->
+:::
```

Comment:
what should we write in this section ? 

--- 

Kuba mentioned on discord: 

I think that right now the icon must be in `client/ayon_applications/icons/{filename}`.
And in future also in `public/icons/{filename}` 

which means that admins must adjust the codebase then create and upload a package  in order to just add an icon.. 
This sounds too much for just adding an icon 😅.

Although, discussing this matter is irrelevant to the doc PR, I think it can help us come up with ideas for writing the doc.

Reply:
Either we write:

> The Application icon must currently live within the codebase and get packaged along with the addon and looks in the addon's client icons folder and the public icons on the server. Note that a different workflow where we can reliably manage these icons and serve them across different sites is still on the backlog.

Or we discard that section for now.

Reply:
`Icon filename. At this moment the icon must live in applications addon codebase. More options will come in future releases.`

Reply:
https://github.com/ynput/ayon-documentation/pull/220/commits/29fc2ae5149c3d4f6cf2aa005f30d453da9593c0